tests/test_predict.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+from pathlib import Path
+
+import src.predict as pr
+import src.file_readers as rd
```

Comment:
This is less readable than just doing `from src import predict` and using `predict.blah` instead of `pr.blah`. You in 2 years will completely forget what `pr` or `rd` mean when reading the code below. Be kind to you in 2 years.

Reply:
Implemented this change in all `.py` files.